bundles/org.openhab.binding.kostalinverter/src/main/resources/ESH-INF/config/SecondGeneration.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0" xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="thing-type:kostalpiko1020_config">
+		<parameter name="url" type="text">
+			<label>Url</label>
+			<description>URL to inverter.</description>
+			<required>true</required>
+		</parameter>
+		<parameter name="username" type="text">
+			<label>Username</label>
+			<description>The username for the inverter.</description>
+			<required>false</required>
+		</parameter>
+		<parameter name="password" type="text">
+			<label>Password</label>
+			<description>The password for the inverter.</description>
+			<required>false</required>
+		</parameter>
+		<parameter name="refreshInterval" type="integer" unit="s">
+			<label>Refresh Interval</label>
+			<description>Refresh Interval in seconds.</description>
+			<required>false</required>
```

Comment:
You might want to specify a default value `<default>10</default>`. Are you sure this is not required?